www/includes/easyparliament/templates/html/alert/_mp_alert_form.php

```diff
@@ -64,7 +64,7 @@
           <input type="text" class="form-control" name="mp_search" id="mp_search" value="<?= _htmlspecialchars($keyword) ?>">
         <?php } else { ?>
           <label for="mp-postcode">Search postcode, or MP name</label>
-          <input id="mp-postcode" type="text" class="form-control" placeholder="<?= gettext('e.g. ‘B2 4QA’ or ‘John Doe’') ?>" name="mp_search" id="mp_search" value="<?= _htmlentities($search_text) ?>" style="min-width:300px;">
+          <input id="mp-postcode" type="text" class="form-control" placeholder="<?= gettext('e.g. ‘B2 4QA’ or ‘John Doe’') ?>" name="mp_search" id="mp_search" value="<?= _htmlentities($search_term) ?>" style="min-width:300px;">
```

Comment:
If I submit with this filled in and no email, the page now gives an error, but this is now blank. Not important, but just checking this was the right thing put here.

Reply:
(I think the setting this back to search_text might have been put in the wrong commit, the gettext one, but assume this will all be squashed/tidied anyway).